src/core/worker.js

```diff
@@ -33,12 +34,17 @@ var WorkerMessageHandler = PDFJS.WorkerMessageHandler = {
         var numPagesPromise = pdfManager.ensureDoc('numPages');
         var fingerprintPromise = pdfManager.ensureDoc('fingerprint');
         var encryptedPromise = pdfManager.ensureXRef('encrypt');
+        var linearizedPromise = pdfManager.ensureXRef('linearization');
+        var startXRefPromise = pdfManager.ensureXRef('startXRef');
```

Comment:
A question: Is it actually necessary to add these promises, since their results don't appear to be used anywhere in `api.js`?